template-only-bin/destroy-app-service.sh

```diff
@@ -8,6 +8,10 @@ TF_VARS_FILE="dev.tfvars"
 
 cd infra/app/service
 
+sed -i.bak 's/resource "aws_s3_bucket" "load_balancer_logs" {/&\n  force_destroy = true/' ../../infra/modules/service/main.tf
```

Comment:
looks like you have the relative path wrong. might be simpler to do this before the `cd infra/app/service`
```suggestion
sed -i.bak 's/resource "aws_s3_bucket" "load_balancer_logs" {/&\n  force_destroy = true/' infra/modules/service/main.tf

cd infra/app/service
```